### PR TITLE
Add upload prevention safties for other filetypes

### PR DESCRIPTION
**Pull request by vxbinaca**

### CODE REVIEW

File: tubeup/TubeUp.py [hunk 1]
```diff
@@ -337,6 +337,27 @@ def upload_ia(self, videobasename, custom_meta=None):
...
+            raise Exception(msg)
+        if glob.glob(videobasename + '*.248*'):
+            msg = 'Video download incomplete, re-attempt archival attempt, exiting...'
+            raise Exception(msg)
+        if glob.glob(videobasename + '*.f247*'):
+            msg = 'Video download incomplete, re-attempt archival attempt, exiting...'
+            raise Exception(msg)
+        if glob.glob(videobasename + '*.temp'):
+            msg = 'Video download incomplete, re-attempt archival attempt, exiting...'
+            raise Exception(msg)
```
Comment (mrpapersonic) [inline tubeup/TubeUp.py:360]:
```suggestion
        for ext in ['*.part', '*.f303*', '*.f302*', '*.ytdl', '*.f251*', '*.248*', '*.f247*', '*.temp']:
            if glob.glob(videobasename + ext):
                msg = 'Video download incomplete, re-attempt archival attempt, exiting...'
                raise Exception(msg)
```

**Comment by vxbinaca** [comment]
Thanks @mrpapersonic you took my code and make it much more compact and readable, probably actually made it work too.

**Comment by vxbinaca** [comment]
> @vxbinaca sorry for the review delay, will get to this shortly 👍

We all have lives it's okay.

**Comment by vxbinaca** [comment]
@mrpapersonic I'll be the humble guy and give you full credit for that snipet when we push a new version. You did much better than me.

**Comment by vxbinaca** [comment]
> 

yt-dlp has safties for say, live videos. Temporary files being left can be handled by the user by clearing the `~/.tubeup/downloads/` folder. We could impliment this on our end, however you'd want to invoke clearing yt-dlps cache thats in a separate location too.

**Pull request merged**